src/signup-form/de/smallTexts.yml

```diff
@@ -0,0 +1,84 @@
+comment: This yaml file is not used by the website. It's a copy of the version that is on contentful
```

Comment:
What's the point of the copied German Yaml files in the `us/` folder? They shouldn't be in this PR I'm guessing?

Reply:
the small texts yaml file in the us folder is in English. The signup form yaml file is not in English, but is required, otherwise the signup page is not shown on the English page. It still needs to be translated, but that will be Ingo's job.